api/messages.go

```diff
@@ -43,6 +43,20 @@ type CreateCredentialResponse struct {
 	ExpiresAt int64  `json:"expiresAt"`
 }
 
+type OperatorInfoRequest struct {
+	Address string `json:"address"`
+	Msg     string `json:"msg"`
+	Sig     string `json:"sig"`
+	Version string `json:"version"`
+
+	operatorType credentials.OperatorType `json:"-"`
+}
```

Comment:
```suggestion
// OperatorInfoRequests have the same form as credential requests, so use a type alias to avoid duplicate code.
type OperatorInfoRequest CredentialRequest
```